flask/helpers.py

```diff
@@ -56,10 +56,20 @@
                     if sep not in (None, '/'))
 
 
-def get_debug_flag(default=None):
+def get_env():
+    val = os.environ.get('FLASK_ENV')
+    if not val:
+        val = 'production'
+    return val
+
+
+def get_debug_flag():
     val = os.environ.get('FLASK_DEBUG')
     if not val:
-        return default
+        env = get_env()
+        if env == 'development':
```

Comment:
`return env == 'development'`? ;)

Also, I'd support shortcuts like `dev`

Reply:
I want to stay compatible with node here.